pkg/cmd/list.go

```diff
@@ -59,85 +67,107 @@ func NewCmdWhoCan() *cobra.Command {
   kubectl who-can create services -n foo
 
   # List who can get the service named "mongodb" in namespace "bar"
-  kubectl who-can get svc mongodb --namespace bar`,
+  kubectl who-can get svc mongodb --namespace bar
+
+  # List who can read pod logs
+  kubectl who-can get pods --subresource=log`,
```

Comment:
Is this consistent with how can-i does it? And do we also support something like "who-can get pods/log"?

Reply:
Yes. The `can-i` subcommand has also the `--subresource` flag. We do not support `who-can get pods/logs`. You have to explicitly specify the subresource via the `--subresource` flag.

```
$ kubectl auth can-i get pods --subresource=log --as dpacak
yes
$ kubectl auth can-i get pods/logs --as dpacak
no
$ kubectl auth can-i get pods/log --as dpacak
no
$ kubectl who-can get pods --subresource=log
ROLEBINDING              NAMESPACE  SUBJECT  TYPE  SA-NAMESPACE
dpacak-can-get-pod-logs  default    dpacak   User

...
```

NB For `can-i` the invocation `kubect auth can-i get pod/logs` would mean check if I can get `pod` named `logs`. Whereas in `who-can` we have a third arg to specify the resource name. That is somehow inconsistent between the two.

Reply:
Interesting, I suppose we should get ourselves consistent with `can-i` for this `<resource type>/<resource name` invocation. But that can be a separate issue / PR.